.github/workflows/deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: Deploy to PyPi
```

Comment:
perhaps just "Deploy" or "CD" if we want to provide also a conda package in the future?